nava/functions.py

```diff
@@ -112,46 +87,37 @@ def __play_winmm(sound_path, async_mode=False, loop=False):
         __play_winmm_flags(sound_path, async_mode, loop)
 
 
-def __play_winmm_flags(sound_path, async_mode=False, loop=False):
+def __play_winmm_flags(sound_path: str, async_mode: bool = False, loop: bool = False) -> None:
     """
     Play a sound using winmm with optional looping.
 
     :param sound_path: sound path
-    :type sound_path: str
```

Comment:
Add `:param async_mode: async mode flag`

Reply:
fixed in e0c85cb.